node-operators/validators/onboarding/register-in-symbiotic.md

```diff
@@ -0,0 +1,183 @@
+
+## Checking Prerequisites {: #checking-prerequisites }
+
+Before registering as an operator, ensure you have already [set up the node](/node-operators/validators/onboarding/run-a-validator/){target=\_blank} and it is running.
+
+To follow this guide, you must interact with smart contracts.
+--8<-- 'text/node-operators/validators/onboarding/run-a-validator/prerequisites.md'
+
+## Register as an Operator {: #registering-operator }
+
+The Symbiotic protocol keeps a registry of all operators. Before being able to secure Tanssi-powered networks, node operators must register as operators in the Symbiotic protocol using one of the methods described in the following sections.
+
+### Register Using the Symbiotic CLI {: #register-with-cli }
+
+If you correctly installed the [Symbiotic CLI](#set-up-the-cli) and you want to sign the transaction using a Ledger device, then run the following command, replacing the `INSERT_OPERATOR_ADDRESS` with your account:
+
+=== "MainNet"
+
+    ```bash
+    python3 symb.py register-operator --ledger --ledger-account INSERT_OPERATOR_ADDRESS
+    ```
+
+=== "TestNet (Sepolia)"
+
+    ```bash
+    python3 symb.py register-operator --ledger --ledger-account INSERT_OPERATOR_ADDRESS --chain sepolia
+    ```
+
+If you want to sign the transaction directly using the account private key, then run the following command, replacing the `INSERT_PRIVATE_KEY` parameter:
+
+=== "MainNet"
+
+    ```bash
+    python3 symb.py register-operator --private-key INSERT_PRIVATE_KEY
+    ```
+
+=== "TestNet (Sepolia)"
+
+    ```bash
+    python3 symb.py register-operator --private-key INSERT_PRIVATE_KEY --chain sepolia
+    ```
+
+!!! warning
+    Note that this method requires you to expose your private key; therefore, it is not recommended.
+
+### Register Using Etherscan {: #register-with-etherscan }
+
+You can interact with Symbiotic's smart contracts using Etherscan and sign the transaction using a browser wallet ([MetaMask](/builders/toolkit/ethereum-api/wallets/metamask/#install-the-metamask-extension){target=\_blank}, for example). 
+
+Go to the contract's page by opening the link:
+
+=== "MainNet"
+
+    [Contract address: {{ networks.symbiotic.contracts.mainnet.operators_registry }}](https://etherscan.io/address/{{ networks.symbiotic.contracts.mainnet.operators_registry }}#writeContract){target=\_blank}
+
+=== "TestNet (Sepolia)"
+
+    [Contract address: {{ networks.symbiotic.contracts.sepolia.operators_registry }}](https://sepolia.etherscan.io/address/{{ networks.symbiotic.contracts.sepolia.operators_registry }}#writeContract){target=\_blank}
+
+Click on **Connect to Web3**, and select your preferred wallet (e.g. MetaMask):
+
+![Connect to Web3 step](/images/node-operators/validators/onboarding/register-in-symbiotic/register-in-symbiotic-1.webp)
+
+!!! note
+    You can configure MetaMask to use a cold wallet.
+
+Once connected:
+
+1. Expand the `registerOperator` function
```

Comment:
```suggestion
1. Expand the **`registerOperator`** function
```

Even though it's a function, it's still an element presented on the UI, so should be bold